Units/parser-python.r/python-import.d/expected.tags

```diff
@@ -2,42 +2,42 @@ a	input.py	/^import a$/;"	module	roles:imported
 c	input.py	/^import c, d$/;"	module	roles:imported
 d	input.py	/^import c, d$/;"	module	roles:imported
 e	input.py	/^from e import f$/;"	module	roles:namespace
-f	input.py	/^from e import f$/;"	unknown	roles:imported
-e.f	input.py	/^from e import f$/;"	unknown	roles:imported
+f	input.py	/^from e import f$/;"	unknown	module:e	roles:imported
```

Comment:
The same. How about add `scope` field for this test?

Reply:
"scope:" field prefix is put only when `--fields=+Z` is given. Z is not given by default.